code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.tsx

```diff
@@ -37,19 +39,43 @@ const OrdersTableBody = ({
     createdAt,
     total,
     orderStatus,
+    isPaid,
+    availableStatuses,
     receiver: { firstName, lastName, email },
-    postAddress: { deliveryMethod },
-    isPaid
+    postAddress: { deliveryMethod }
   } = order;
 
+  const user = useUserDetailsSelector();
+
   const orderReceiver = `${lastName} ${firstName}`;
 
+  const userId = user?.id;
+  const userRole = user?.role;
+  const loginWithAdminRole = Boolean(userRole === ROLES.ADMIN && userId);
+  const loginWithManagerRole = Boolean(
+    userRole === ROLES.SHOP_MANAGER && userId
+  );
+
+  const filteredOrderStatuses: OrderStatus[] = availableStatuses.length
```

Comment:
I have a suggestion:

```suggestion
  const filteredOrderStatuses: OrderStatus[] = availableStatuses.length > 0
```

It would be more readable and logical in my opinion

Reply:
I also used this approach before, but this line wasn't covered by stryker and I changed it (variant with boolean also didn;t work)))
I have changed it back , if it is more readable, as you say!